components/freertos/ringbuf.c

```diff
@@ -409,6 +409,16 @@ static void returnItemToRingbufBytebuf(ringbuf_t *rb, void *item) {
     //Free the read memory.
     rb->free_ptr=rb->read_ptr;
 }
+/*
+    find out if semaphore returned from xQueueSelectFromSet belong to this xRingbuffer
+*/
+BaseType_t xRingbufferCanRead(RingbufHandle_t ringbuf, QueueHandle_t sem)
```

Comment:
Suggest changing from ``QueueHandle_t sem`` to ``QueueSetMemberHandle_t member`` to conform with FreeRTOS queue set API